openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EntityRepository.java

```diff
@@ -1344,6 +1389,7 @@ public PutResponse<T> restoreEntity(String updatedBy, String entityType, UUID id
     updated.setUpdatedAt(System.currentTimeMillis());
     EntityUpdater updater = getUpdater(original, updated, Operation.PUT);
     updater.update();
+    restoreFromSearch(updated);
```

Comment:
@harshach , here i am calling this from entity resource as in entity repository it is recursive function call to restore the children